client/src/app/site/pages/meetings/pages/projectors/modules/projector-detail/components/projector-detail/projector-detail.component.ts

```diff
@@ -66,6 +69,21 @@ export class ProjectorDetailComponent extends BaseMeetingComponent implements On
      */
     public editQueue = false;
 
+    public get hasSlide(): boolean {
+        return !!this.projector.current_projections.filter(projection => projection.stable === false).length;
```

Comment:
You could use `this.projector.nonStableCurrentProjections` here and in the following occurrences of filtering non stable projections.